packages/cubejs-schema-compiler/adapter/BaseQuery.js

```diff
@@ -952,17 +952,35 @@ class BaseQuery {
           this.safeEvaluateSymbolContext().leafMeasures[this.safeEvaluateSymbolContext().currentMeasure] = true;
         }
       }
+      let auto_prefix;
+      if (this.safeEvaluateSymbolContext().usedInAggregationWithFilter){
+        // TODO - check if this is a measure in the cube
+        // TODO - function for commonly setting this
+        // TODO - confirm context
+        // TODO - don't return here, return later
+        return this.escapeColumnName(`${cubeName}__${name}`)
```

Comment:
There's got to be an existing function for this. Unable to determine how. Should I make a function for this?

Reply:
probably shouldn't return here